includes/class-wc-payments-account.php

```diff
@@ -756,4 +756,20 @@ private function get_actioned_notes(): array {
 
 		return $wcpay_note_names;
 	}
+
+	/**
+	 * Gets the account country.
+	 *
+	 * @return string|null Country.
+	 */
+	public function get_account_country() {
+		$account = $this->get_cached_account_data();
+
+		// When there is an account connected, if 'country' is missing, default to US.
+		if ( $account ) {
+			return $account['country'] ?? 'US';
+		}
+
+		return null;
```

Comment:
Following up on my comment with an injection of the country via a parameter, we can fallback to 'US' here as well. Perhaps we can compact the method nicely here =)

Reply:
Done. Like so?

Reply:
Yes, thanks!